cpp/src/aztec/plonk/composer/standard_composer.cpp

```diff
@@ -743,6 +741,7 @@ std::shared_ptr<proving_key> StandardComposer::compute_proving_key()
 
     circuit_proving_key->recursive_proof_public_input_indices =
         std::vector<uint32_t>(recursive_proof_public_input_indices.begin(), recursive_proof_public_input_indices.end());
+    // What does this line do exactly?
```

Comment:
Note to self: `contains_recursive_proof` was removed from the proving key iirc. Check this.